src/components/profile-modal.jsx

```diff
@@ -13,9 +14,14 @@ const CloseButton = ({ onClose }) => (
   </button>
 );
 
-const ProfileModal = ({ mentor, onClose }) => {
+const ProfileModal = ({ isModalActive, mentor, onClose }) => {
```

Comment:
If isModalActive is only used here, why is there a difference between isModalActive and isModalOpen?